test/katello_test_helper.rb

```diff
@@ -88,7 +88,7 @@ module FixtureTestCase
 
     Setting::Content.load_defaults
 
-    @@admin = ::User.find(FIXTURES['users']['admin']['id'])
+    @@admin = ::User.unscoped.find(FIXTURES['users']['admin']['id'])
```

Comment:
Still failing: 
01:56:22.484 /var/lib/workspace/workspace/test_katello_pull_request/slave/fast/plugin/test/katello_test_helper.rb:65:1: W: Unnecessary disabling of Style/ClassVars.
01:56:22.484 # rubocop:disable Style/ClassVars